velox/exec/tests/AggregationTest.cpp

```diff
@@ -988,6 +988,7 @@ DEBUG_ONLY_TEST_F(AggregationTest, spillWithEmptyPartition) {
             .config(
                 QueryConfig::kSpillStartPartitionBit,
                 std::to_string(kPartitionStartBit))
+            .config(QueryConfig::kPreferredOutputBatchBytes, "1024")
```

Comment:
Do we need to specify this since it is the default?

Reply:
The default value of kPreferredOutputBatchBytes is 10MBs. Here is to set a small output batch size to remove a memory limit error.